### PR TITLE
Do not fail when a scenario cannot be simulated, but show a warning

### LINKED ISSUE #493: Show informative error when a scenario fails
When a scenario simulation fails, show which did fail.

### PR DESCRIPTION
**Pull request by PavelBal**
Fixes #493

### CODE REVIEW

**Comment by PavelBal** [comment]
I also tried to bring some structure to the messages file.

File: R/messages.R [hunk 1]
```diff
@@ -27,52 +30,50 @@ messages$errorWrongPopulationName <- function(populationName) {
...
 }
 
+# utilities-individual####
 messages$errorWrongIndividualId <- function(individualId) {
   paste0("Individual with id ", individualId, " is not specified in the individual characteristics file!")
 }
 
+# utilities####
 messages$warningValueWithinThresholdNotExisting <- function(value, threshold, optionalMessage = NULL) {
   paste(
```
Comment (Felixmil) [inline R/messages.R:40]:
```suggestion
  paste0(
```

**Review by Felixmil — CHANGES_REQUESTED** [review]
That's way better like this.

I spoted a `paste()` instead of `paste0()` and also maybe it should be harmonized in terms of indentations:
```r
paste0("....
        .....')

# or 

paste0(
  ".....
   ......"
)
```

Another improvement suggestion would be to use `glue::glue()` for easier readability. It is not in DESCRIPTION dependency list right now but we already have it indirectly since it's a ggplot2 dependency (and many other packages we use).

**Comment by PavelBal** [comment]
> Another improvement suggestion would be to use `glue::glue()` for easier readability.

Let's do it separately (feel free to implement, I have not used `glue` so far ;) )

**Review by Felixmil — APPROVED** [review]

**Pull request merged**